blueprint/api.apib

```diff
@@ -2248,7 +2248,7 @@ This endpoint allows you to check whether a username is valid (by running a vali
 ## Stripe Plan Attributes (object)
 + amount: 999 (number) - Describes the amount of a plan in cents.
 + created: `2016-07-08T03:03:51.967Z` (string)
-+ id_from_stripe: `monthly-donation-1`
++ id-from-stripe: `monthly-donation-1`
```

Comment:
I think this needs to be surrounded with ticks. Is this rendering for you without doing that?